gno.land/pkg/gnoland/genesis.go

```diff
@@ -109,11 +109,83 @@ func LoadGenesisParamsFile(path string, ggs *GnoGenesisState) error {
 		}
 	}
 
+	// Write onto ggs.Bank.Params.
```

Comment:
Thank you for adding the missing genesis loading. However, I believe we should proceed in the following direction:

-  Implementing the local switch case into each parameter, as shown in https://github.com/gnolang/gno/blob/f9e4457cb242b1cd29fbdd199c15d79c53f71bcb/tm2/pkg/sdk/bank/params.go#L62-L69
-  Refactoring this genesis function so that it simply calls WillSetParam for each parameter.